sdk/communication/communication-phone-numbers/tests.yml

```diff
@@ -5,11 +5,10 @@ stages:
     parameters:
       PackageName: "@azure/communication-phone-numbers"
       ServiceDirectory: communication
-      EnvVars:
-        AZURE_SUBSCRIPTION_ID: $(acs-subscription-id)
-        AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING: $(communication-livetest-connection-string)
-        AZURE_PHONE_NUMBER: $(js-communication-livetest-phone-number)
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        COMMUNICATION_SAMPLES_CONNECTION_STRING: $(communication-livetest-connection-string)
+      CloudConfig:
+        Public:
+          SubscriptionConfigurations:
+            - $(sub-config-azure-cloud-test-resources)
+            - $(sub-config-communication-services-cloud-test-resources-test)
```

Comment:
this is renamed to sub-config-communication-services-cloud-test-resources-common